polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -459,17 +472,34 @@ def set_ee_pose(
 
         return self.send_torch_policy(torch_policy=torch_policy, **kwargs)
 
+    """
+    PyRobot backward compatibility methods
+    """
```

Comment:
- Print an error message saying these are deprecated? e.g. `log.warn` and suggest the new method name
- Add docstring with bolded deprecation warning (and then make sure to regenerate the documentation)

Reply:
Should we be deprecating them right away if we're still advocating Polymetis as a drop-in replacement for PyRobot?
It doesn't make sense if a user replaces PyRobot calls with Polymetis and ends up with a bunch of deprecation messages.
However, if not many people are affected then I think we can deprecate them. Do you know if there are any modules (maybe in droidlet) that switched to Polymetis from PyRobot?

Reply:
Droidlet has not switched from PyRobot to Polymetis, though tbh PyRobot is unmaintained and doesn't seem to have been touched since Zisu left. You can ping the droidlet team to be sure, but in general I am fine with adding deprecation messages.

Reply:
Sounds good, I'll just deprecate them.

Reply:
Also, I thought the documentation auto-regenerate & upload in CircleCI?

Reply:
Not the manually-written rst files -- not sure if any needs to be updated.